sdk/api_readme.md

```diff
@@ -6,9 +6,10 @@ Welcome to the Dart API reference documentation, covering the
   * [dart:core](dart-core/dart-core-library.html): Core functionality such as
     strings, numbers, collections, errors, dates, and URIs.
   * [dart:io](dart-io/dart-io-library.html): I/O for non-web apps.
-  * [dart:html](dart-html/dart-html-library.html): DOM manipulation for web apps
-    (available only to web apps).
-
+  * [dart:js_interop](dart-js_interop/dart-js_interop-library.html)
+    and [package:web](https://pub.dev/documentation/web):
+    Web interop features and APIs for accessing JavaScript and browser bindings.
+    
```

Comment:
I think it's slightly the other way around, as the APIs in these libraries are bindings to the underlying JS and web APIs. Would something like the following work?

```suggestion
  * [dart:js_interop](dart-js_interop/dart-js_interop-library.html)
    and [package:web](https://pub.dev/documentation/web):
    Web interop features and API bindings for accessing JavaScript and web libraries.

```

Reply:
For extra context on why I changed "browser" to "web" is because developers might understand "browser" to also mean browser extensions. While those are possible using `dart:js_interop`, that doesn't seem to be the goal of this text :)

Reply:
Oh that makes sense! Thanks for explaining